src/main/java/frc/robot/RobotContainer.java

```diff
@@ -44,7 +47,7 @@ public class RobotContainer {
   private final Claw claw = new Claw();
 
   private final DriverController mainController = new DriverController(Constants.DeviceIds.driverController);
-
+  private final DriverController alternaController = new DriverController(Constants.DeviceIds.driverController);
```

Comment:
You used the controller id for the `mainController`, you need to create a new one for the `alernateController`.

Also, you misspelled alternate.